packages/validator/src/services/prepareBeaconProposer.ts

```diff
@@ -51,4 +75,64 @@ export class PrepareBeaconProposerService {
       ),
     }));
   }
+
+  private registerValidator = async (epoch: Epoch): Promise<void> => {
+    if (epoch < this.config.BELLATRIX_FORK_EPOCH - 1) return;
+    this.getValidatorRegistrations(epoch, this.indicesService.getAllLocalIndices())
```

Comment:
why do we have to register all validators for every epoch?

Reply:
because a validator registration might only be valid for 2 epochs on the builder, its just simpler to blindly register validator with them every epoch. :thinking: